metadata-service/factories/src/main/resources/application.yml

```diff
@@ -80,6 +80,7 @@ elasticsearch:
   username: ${ELASTICSEARCH_USERNAME:#{null}}
   password: ${ELASTICSEARCH_PASSWORD:#{null}}
   pathPrefix: ${ELASTICSEARCH_PATH_PREFIX:#{null}}
+  mainTokenizer: ${ELASTICSEARCH_MAIN_TOKENIZER:#{null}}
```

Comment:
nit: can we put this under index ? Been trying to put all index builder related variables into index

Reply:
@dexter-mh-lee Of course, no problem.

Reply:
@Huyueeer When you get a chance to do this, we will merge it!

Reply:
@dexter-mh-lee @shirshanka OK. I'm sorry for the delay. I've made changes.